packages/react-tinacms-github/src/toolbar-plugins/createMarkdownDeleteAction.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+import {
+  ActionButton,
+  useCMS,
+  Form,
+  Modal,
+  ModalPopup,
+  ModalHeader,
+  ModalBody,
+  ModalActions,
+} from 'tinacms'
```

Comment:
I would prefer to remove the `tinacms` dependency and load these from the core packages directly. Something like this:

```
import { Modal, ... } from "@tinacms/modals"
import { Form, ActionButton } from "@tinacms/react-forms"
import { useCMS } from "@tinacms/react-core
```

Reply:
Should be fixed now